Tests/run_tests.py

```diff
@@ -254,10 +256,13 @@ def main(argv):
             return 0
         if opt == "--offline":
             print("Skipping any tests requiring internet access")
-            # This is a bit of a hack...
-            import requires_internet
-            requires_internet.check.available = False
-            # The check() function should now report internet not available
+            # Monkey patch for urlopen()
+            import Bio._py3k
+
+            def dummy_urlopen(url):
+                raise MissingExternalDependencyError("internet not available")
```

Comment:
I think this should be a runtime error - if this happens it means the test did not respect the ``requires_internet`` which (assuming that bit of code remains above).